.github/workflows/cross-compiles.yml

```diff
@@ -97,6 +97,12 @@ jobs:
             libs: libc6-dev-sh4-cross,
             target: no-async linux-latomic,
             tests: -test_includes -test_store -test_x509_store
+          }, {
+            arch: sparc64-linux-gnu,
```

Comment:
We have the same arch already down there, we just do not run any tests for it.

Reply:
The cpu capabilities is different, but I agree the no test version could be removed.

Reply:
But this is in line with lots of other targets, where a -static version excluding engines etc. is built for the sole purpose to have something that can be executed by qemu without crashing immediately. And then we have a shared version built again, just to see that the engines etc. can compile for that target without errors and warnings.
So IMO that is not wasted.
So I would agree that no test version should be removed, just this one added :grinning:

Reply:
We have too many CI runs already. The only difference is that this is a -static build. All other cross-compile builds are actually non-static.